package.json

```diff
@@ -24,7 +24,7 @@
     "trailingComma": "none"
   },
   "peerDependencies": {
-    "eslint-plugin-typescript": "*",
-    "typescript-eslint-parser": "*"
+    "@typescript-eslint/eslint-plugin": "^1.6.0",
```

Comment:
Another thing I forgot was that this package's job is to ask for the latest versions of these packages as peerDeps, so please restore the `*`

Reply:
done